pkg/cli/options_test.go

```diff
@@ -34,6 +35,167 @@ import (
 )
 
 var _ = Describe("Discover external plugins", func() {
+	Context("with valid plugins root path", func() {
+		var (
+			homePath           string = os.Getenv("HOME")
+			customPath         string = "/tmp/myplugins"
+			xdghome            string = os.Getenv("XDG_CONFIG_HOME")
+			originalPluginPath string = os.Getenv("KB_PLUGINS_PATH")
```

Comment:
Nit: I'd prefer if we did all the `os.Getenv()` calls in the `BeforeEach` for the appropriate context they are used. This will make sure that we are constantly fetching the most up to date value before a test is run.

Reply:
Oh, I see.
I tried to update the code following your suggestions. It seems much clearer than before!